### ISSUE TITLE
Typo in loss calculation

### ISSUE THREAD

**Issue by vinsis** [yifjiang/relative-depth-using-pytorch#7]
Hi Yifan, there seems to be a typo in [this line](https://github.com/yifjiang/relative-depth-using-pytorch/blob/master/models/criterion/relative_depth.py#L11). It should be `z_B` instead of `a_B`.

Just wondering - how did the entire thing work with this typo in place? Or is this piece of code never used in actual training?

**Comment by yifjiang** [comment]
You are right. This is definitely a typo.
However, this piece of code is used in actual training. The reason it still works is because Broadcasting rules are applied when `z_A` and `z_B` are of different shape.
Basicially, [this line](https://github.com/yifjiang/relative-depth-using-pytorch/blob/520563e2ff31a724e07146afecefceb4a835f8d7/models/criterion/relative_depth.py#L10) and the following line only unwrap the tensors. And in [this line](https://github.com/yifjiang/relative-depth-using-pytorch/blob/520563e2ff31a724e07146afecefceb4a835f8d7/models/criterion/relative_depth.py#L12) broadcasting rule is applied and `z_A` is converted back to the wrapped version again.
Fortunately, the additional dimension doesn't affect the result since everything becomes a scalar because of the `sum` operation in [this line](https://github.com/yifjiang/relative-depth-using-pytorch/blob/520563e2ff31a724e07146afecefceb4a835f8d7/models/criterion/relative_depth.py#L36).
Anyway, thanks for pointing this out. I will fix it later.

**Comment by vinsis** [comment]
Thanks for answering. A couple of quick questions before I close this issue:

- Do you resize the image to (width, height) = (320, 240) before feeding to the model?
- Did you train the model from scratch only using the DIW dataset (with no NYU v2 dataset)? If so, can you point me to where it's done? It's a little hard to follow without any documentation.

Thanks again.
